modules/sovrnBidAdapter.js

```diff
@@ -91,67 +91,63 @@ var SovrnAdapter = function SovrnAdapter() {
 
   // expose the callback to the global object:
   $$PREBID_GLOBAL$$.sovrnResponse = function (sovrnResponseObj) {
-    // valid object?
-    if (sovrnResponseObj && sovrnResponseObj.id) {
-      // valid object w/ bid responses?
-      if (sovrnResponseObj.seatbid && sovrnResponseObj.seatbid.length !== 0 && sovrnResponseObj.seatbid[0].bid && sovrnResponseObj.seatbid[0].bid.length !== 0) {
-        var impidsWithBidBack = [];
-        sovrnResponseObj.seatbid[0].bid.forEach(function (sovrnBid) {
-          var responseCPM;
-          var placementCode = '';
-          var id = sovrnBid.impid;
-          var bid = {};
-
-          // try to fetch the bid request we sent Sovrn
-          var bidObj = $$PREBID_GLOBAL$$._bidsRequested.find(bidSet => bidSet.bidderCode === 'sovrn').bids
-            .find(bid => bid.bidId === id);
-
-          if (bidObj) {
-            placementCode = bidObj.placementCode;
-            bidObj.status = CONSTANTS.STATUS.GOOD;
-
-            // place ad response on bidmanager._adResponsesByBidderId
-            responseCPM = parseFloat(sovrnBid.price);
-
-            if (responseCPM !== 0) {
-              sovrnBid.placementCode = placementCode;
-              sovrnBid.size = bidObj.sizes;
-              var responseAd = sovrnBid.adm;
-
-              // build impression url from response
-              var responseNurl = '<img src="' + sovrnBid.nurl + '">';
-
-              // store bid response
-              // bid status is good (indicating 1)
-              bid = bidfactory.createBid(1, bidObj);
-              bid.creative_id = sovrnBid.id;
-              bid.bidderCode = 'sovrn';
-              bid.cpm = responseCPM;
-
-              // set ad content + impression url
-              // sovrn returns <script> block, so use bid.ad, not bid.adurl
-              bid.ad = decodeURIComponent(responseAd + responseNurl);
-
-              // Set width and height from response now
-              bid.width = parseInt(sovrnBid.w);
-              bid.height = parseInt(sovrnBid.h);
-
-              bidmanager.addBidResponse(placementCode, bid);
-              impidsWithBidBack.push(id);
+    var impidsWithBidBack = [];
+
+    // valid response object from sovrn
+    if (sovrnResponseObj && sovrnResponseObj.id && sovrnResponseObj.seatbid && sovrnResponseObj.seatbid.length !== 0 &&
+      sovrnResponseObj.seatbid[0].bid && sovrnResponseObj.seatbid[0].bid.length !== 0) {
+      sovrnResponseObj.seatbid[0].bid.forEach(function (sovrnBid) {
+        var responseCPM;
+        var placementCode = '';
+        var id = sovrnBid.impid;
+        var bid = {};
+
+        // try to fetch the bid request we sent Sovrn
+        var bidObj = $$PREBID_GLOBAL$$._bidsRequested.find(bidSet => bidSet.bidderCode === 'sovrn').bids
```

Comment:
These private variables like `pbjs._bidsRequested` will be going away with the implementation of concurrent auctions in #1421 

Something to keep in mind. However, I won't make it a requirement for this pull-request to not use them since it's not changing the existing functionality of the adapter (which was already using them).  But it might be worthwhile to move away from their usage going forward.